src/utils/validation.ts

```diff
@@ -86,6 +86,9 @@ export const validationRules = {
     tokenCode: allRules.chain,
     tpid: allRules.tpid,
   },
+  transferLockedTokensRequest: {
+    tpid: allRules.tpid,
+  },
```

Comment:
```
  transferLockedTokensRequest: {
    tpid: allRules.tpid,
    payeePublicKey: allRules.fioPublicKey,
  },
```

Reply:
same comment the fioPublicKey validator isn't working presently, will go back and check it out..

Reply:
let's leave as was before for now 
```
 transferLockedTokensRequest: {
    tpid: allRules.tpid,
  },
```

Reply:
we will resolve this in a follow on story

Reply:
we will do it in a future story.